docs/docs/examples/guides/camera-calibration.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 9
+title: Camera calibration walkthrough
+---
+
+import cameraCalibration from './assets/camera-calibration.gif'
+
+# Camera calibration
+
+Accurate camera calibration is essential for ensuring that visual detections and measurements correspond correctly to
+real-world geometry. In many perception pipelines, critical output is expressed in image pixel coordinates. However, if
+your camera's lens introduces distortion or if the intrinsic parameters are unknown, these pixel coordinates can deviate
+significantly from their true positions.
```

Comment:
```suggestion
significantly from their true position.
```
I would have said position singularity but I'm not sure what is gramatically correct